src/test/resources/com/znsio/teswiz/features/vodqa.feature

```diff
@@ -0,0 +1,10 @@
+@vodqa
+Feature: vodqa feature
+
+#  CONFIG=./configs/vodqa_local_config.properties PLATFORM=android TAG=vodqa ./gradlew run
+  @android
+  Scenario: User is able to tap in the middle of the screen
+    Given I login into the application
+    When I tap in the middle of the screen
+    Then I am able to view "Drag & Drop" page
```

Comment:
Don't you think, every device has different screen size and they render the screen accordingly.
for eg. In the particular device you are considering the middle of the screen is "Drag & Drop" but for other device the middle could be something different.

Suggestion: Instead of verifying which section is opening, look for whether you are able to click on middle and be it any section which needs to be open